src/devices/technicmediumangularmotor.ts

```diff
@@ -10,8 +10,8 @@ import * as Consts from "../consts";
  */
 export class TechnicMediumAngularMotor extends AbsoluteMotor {
 
-    constructor (hub: IDeviceInterface, portId: number, modeMap: {[event: string]: number} = {}, type: Consts.DeviceType = Consts.DeviceType.TECHNIC_MEDIUM_ANGULAR_MOTOR) {
-        super(hub, portId, {}, type);
+    constructor (hub: IDeviceInterface, portId: number) {
+        super(hub, portId, [], Consts.DeviceType.TECHNIC_MEDIUM_ANGULAR_MOTOR);
```

Comment:
Other similar motor are defined this way so I though it would be better as it is shorter